testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/Configurations.java

```diff
@@ -87,7 +88,10 @@ public static String nameForConfigClass(Class<? extends EnvironmentConfig> clazz
 
     public static String nameForSuiteClass(Class<? extends Suite> clazz)
     {
-        return CaseFormat.UPPER_CAMEL.to(CaseFormat.LOWER_HYPHEN, clazz.getSimpleName().replace("Suite", "Suite-")).replace("--", "-");
+        String className = clazz.getSimpleName();
+        checkArgument(className.matches("^Suite[A-Z0-9].*"), "Name of %s should start with 'Suite'", clazz);
+        // For a suite "Suite1", the UPPER_CAMEL to LOWER_HYPHEN conversion won't insert a hyphen after "Suite"
+        return "suite-" + CaseFormat.UPPER_CAMEL.to(CaseFormat.LOWER_HYPHEN, className.replaceFirst("^Suite", ""));
```

Comment:
please add a test for this enforcement (to document what is a good and what is a bad practice).

Reply:
we had a discussion about tests for PTL and decided not to have them in general

note this particular method _is_ unit tested though